enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper/PublicComponentWrapper.tsx

```diff
@@ -14,23 +14,27 @@ export const PublicComponentWrapper = ({
 }) => {
   const loginStatus = useSdkSelector(getLoginStatus);
 
+  let content;
```

Comment:
I would prefer `content` to default to `children` here, as semantically the children is also a content rendered by the public component wrapper.